src/main/java/core/basesyntax/strategy/operation/Handler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.model.Store;
+
+public interface Handler {

```

Comment:
```suggestion
public interface OperationHandler {

```
name's a bit too generic